spec/support/fog_helper.rb

```diff
@@ -0,0 +1,4 @@
+Fog.mock!
+Fog.credentials_path = Rails.root.join('config/fog_credentials.yml')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.